### PR TITLE
Clarifying about planner

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7777

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Office-365-groups.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ By default, users in Microsoft 365 can create and manage groups. For more inform
 
 ## How Microsoft 365 Groups work with Teams
 
-When you create a team, a Microsoft 365 group is created to manage team membership. The group's related services, such as a SharePoint site, Power BI workspace, etc. are created at the same time.
+When you create a team, a Microsoft 365 group is created to manage team membership. The group's related services, such as a SharePoint site, Power BI workspace, etc. (Except Planner) are created at the same time.
```
Comment (JohanFreelancer9) [inline Teams/Office-365-groups.md:32]:
```suggestion
When you create a team, a Microsoft 365 group is created to manage team membership. The group's related services such as a SharePoint site and Power BI workspace (except Planner) are created at the same time.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/Office-365-groups.md (hunk 1 above)
Comment (get-itips) [reply Teams/Office-365-groups.md:32]: I feel that if we remove the "," and replace with "and" we left out other services not explicitly mentioned.

Comment (JohanFreelancer9) [reply Teams/Office-365-groups.md:32]: Hi @get-itips. I suggested the change to remove the abbreviation "etc.", which, in accordance with the MS Style Guide, we should not use in docs. I have edited my suggestion, though. Please let me know what you think.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@get-itips I think it's best not to call out Planner here. While the Planner instance isn't created until you access it from the team or group, the entry point is there and it auto-creates the first time you access it. You don't have to go to Planner and then connect to an existing group. I think calling it out as not being created will generate confusion.

**Comment by get-itips** [comment]
> @get-itips I think it's best not to call out Planner here. While the Planner instance isn't created until you access it from the team or group, the entry point is there and it auto-creates the first time you access it. You don't have to go to Planner and then connect to an existing group. I think calling it out as not being created will generate confusion.

Hello Mike, no problem at all, thanks for your clarification.

**Pull request closed without merging**